libs/category/src/effects/category.effects.spec.ts

```diff
@@ -103,229 +96,61 @@ describe('DaffCategoryEffects', () => {
-
-      beforeEach(() => {
-        actions$ = hot('--a', { a: categoryLoadAction });
+       categoryPageLoadAction = new DaffCategoryLoad({id: stubCategory.id});
+       categoryPageLoadSuccessAction = new DaffCategoryLoadSuccess({
+        category: stubCategory,
+        categoryPageConfigurationState: stubCategoryPageConfigurationState,
+        products: stubProducts
       });
-      
+
+      actions$ = hot('--a', { a: categoryPageLoadAction });
+		});
+
+    describe('when the call to CategoryService is successful', () => {
       it('should dispatch a DaffCategoryLoadSuccess and a DaffProductGridLoadSuccess action', () => {
 				spyOn(daffCategoryDriver, 'get').and.returnValue(of({
 					category: stubCategory,
 					categoryPageConfigurationState: stubCategoryPageConfigurationState,
 					products: stubProducts
 				}));
-        
-        expected = cold('--(ab)', { a: productGridLoadSuccessAction, b: categoryLoadSuccessAction });
+
+        expected = cold('--(ab)', { a: productGridLoadSuccessAction, b: categoryPageLoadSuccessAction });
         expect(effects.loadCategory$).toBeObservable(expected);
       });
     });
 
     describe('when the call to CategoryService fails', () => {
-      
+
       beforeEach(() => {
         const error = 'Failed to load the category';
 				const response = cold('#', {}, error);
 				spyOn(daffCategoryDriver, 'get').and.returnValue(response);
-        const categoryLoadFailureAction = new DaffCategoryLoadFailure(error);
-        actions$ = hot('--a', { a: categoryLoadAction });
-        expected = cold('--b', { b: categoryLoadFailureAction });
+        const categoryPageLoadFailureAction = new DaffCategoryLoadFailure(error);
+        expected = cold('--b', { b: categoryPageLoadFailureAction });
       });
-      
+
       it('should dispatch a CategoryLoadFailure action', () => {
         expect(effects.loadCategory$).toBeObservable(expected);
       });
     });
-	});
-
-	describe('validateFilters', () => {
-		
-		it('should throw an error when an invalid range type format is given', () => {
-			expect(() => {
-				effects.validateFilters([{
-					name: 'price',
-					value: ['invalidFormat'],
-					type: DaffCategoryFilterType.Range
-				}])
-			}).toThrowError('Category range filter is in an invalid format. Should be **-**');
-		});
-
-		it('should throw an error when two filters share the same name', () => {
-			expect(() => {
-				effects.validateFilters([{
-					name: 'name',
-					value: ['value'],
-					type: DaffCategoryFilterType.Equal
-				},
-				{
-					name: 'name',
-					value: ['value2'],
-					type: DaffCategoryFilterType.Equal
-				}
-			])
-			}).toThrowError('More than one category filter of the same name exists. These should' +
-						' be combined into a single filter action with multiple values.');
-		});
-	});
 
-  describe('when CategoryLoadAction is triggered', () => {
-
-    let expected;
-    
     it('should call get category with only an id', () => {
```

Comment:
This test isn't quite correct. It can get a category with all of the other inputs, but they need to be passed through the action instead of through state.

Reply:
I think I fixed this